### ISSUE TITLE
Is it possible to use the wildcards for the parameter?

### ISSUE THREAD

**Issue by iagulevich** [namshi/mockserver#67]
In the documentation, I did not manage to find the wildcards for the parameter.
For example, I want to have the following test case: 
the definition: `GET--type=__&location=__.mock`
the expected behavior: have a response for GET method with any `type` and `location`.

I can contribute to this case.

Thank you in advance.

**Comment by odino** [comment]
Happy to review a PR :)

**Comment by kdkaushikdalvi** [comment]
Yes this is very important feature to have such case for request kind "GET--type=__&location=__.mock"
please help anyone to get works.

**Comment by alelapi** [comment]
Hi, any news about this? It would be very useful
